server/src/Client.hpp

```
@@ -0,0 +1,27 @@
+#pragma once
+
+#include <asio.hpp>
+#include <chrono>
+#include <cstddef>
+#include <cstdint>
+#include "Macro.hpp"
+
+namespace server {
+  struct Client {
+    public:
+      Client(int id) : _player_id(id) {
```

Clang (14.0.6)

[warning] constructor does not initialize these fields: _last_heartbeat, _last_position_update

(cppcoreguidelines-pro-type-member-init)